panels/search/cc-search-panel.c

```diff
@@ -614,7 +614,7 @@ search_providers_discover_ready (GObject *source,
 
   for (l = discover_result->providers; l != NULL; l = l->next)
     {
-      g_autoptr(GFile) provider = l->data;
+      GFile *provider = l->data;
```

Comment:
This seems to have been a regression in the rebase of commit c773723d78, and should be squashed into that next time we rebase. I’ve checked, and the bug doesn’t seem to be present on the `eos3.6` branch.

Reply:
(Now proposed in a separate PR as https://github.com/endlessm/gnome-control-center/pull/175.)